system-addon/content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -135,12 +51,10 @@ export class _CollapsibleSection extends React.PureComponent {
   constructor(props) {
     super(props);
     this.onBodyMount = this.onBodyMount.bind(this);
-    this.onInfoEnter = this.onInfoEnter.bind(this);
-    this.onInfoLeave = this.onInfoLeave.bind(this);
```

Comment:
Were these not actually used by `CollapsibleSection`? Looks like it was added twice but only needed in `Info`?

Reply:
oh weird I didnt even notice that! I guess it was some copy/pasta fail when creating `<Disclaimer/>`

Reply:
oh, I guess this isn't `<Disclaimer/>`. I fail at reading diffs. Yeah, this must've been a fail when I broke `<Info/>` into it's own component or something.